src/lib/openapi/spec/addons-schema.ts

```diff
@@ -8,15 +8,256 @@ export const addonsSchema = {
+                        'feature-stale-on',
+                        'feature-stale-off',
+                        'feature-environment-enabled',
+                        'feature-environment-disabled',
+                        'feature-strategy-remove',
+                        'feature-strategy-update',
+                        'feature-strategy-add',
+                        'feature-metadata-updated',
+                        'feature-variants-updated',
+                        'feature-project-change',
+                    ],
+                },
+                {
+                    name: 'datadog',
+                    displayName: 'Datadog',
+                    description: 'Allows Unleash to post updates to Datadog.',
+                    documentationUrl:
+                        'https://docs.getunleash.io/docs/addons/datadog',
+                    parameters: [
+                        {
+                            name: 'url',
+                            displayName: 'Datadog Events URL',
+                            description:
+                                'Default url: https://api.datadoghq.com/api/v1/events. Needs to be changed if your not using the US1 site.',
+                            type: 'url',
+                            required: false,
+                            sensitive: false,
+                        },
+                        {
+                            name: 'apiKey',
+                            displayName: 'Datadog API key',
+                            placeholder: 'j96c23b0f12a6b3434a8d710110bd862',
+                            description: '(Required) API key from Datadog',
+                            type: 'text',
+                            required: true,
+                            sensitive: true,
+                        },
+                    ],
+                    events: [
+                        'feature-created',
+                        'feature-updated',
+                        'feature-archived',
+                        'feature-revived',
+                        'feature-stale-on',
+                        'feature-stale-off',
+                        'feature-environment-enabled',
+                        'feature-environment-disabled',
+                        'feature-strategy-remove',
+                        'feature-strategy-update',
+                        'feature-strategy-add',
+                        'feature-metadata-updated',
+                        'feature-project-change',
+                        'feature-variants-updated',
+                    ],
+                    tagTypes: [
+                        {
+                            name: 'datadog',
+                            description:
+                                'All Datadog tags added to a specific feature are sent to datadog event stream.',
+                            icon: 'D',
+                        },
+                    ],
+                },
+            ],
```

Comment:
This is what you actually get when you call the endpoint. All this stuff is not documented anywhere else.